src/ssl.c

```diff
@@ -20598,7 +20598,7 @@ size_t wolfSSL_get_client_random(const WOLFSSL* ssl, unsigned char* out,
 
 
 #if (defined(KEEP_PEER_CERT) && defined(SESSION_CERTS)) || \
-                                   (defined(OPENSSL_ALL) && defined(HAVE_PKCS7))
+    (defined(OPENSSL_ALL) && defined(SESSION_CERTS))
```

Comment:
Can you explain the change from `HAVE_PKCS7` to `SESSION_CERTS`? Also, the `#endif` comment doesn't match now:

```
#endif /* (KEEP_PEER_CERT && SESSION_CERTS) || (OPENSSL_ALL && HAVE_PKCS7) */
```

Reply:
DecodeToX509() is only called in two places in ssl.c.  Once in the case of `KEEP_PEER_CERT & SESSION_CERTS` and once in the case of `OPENSSL_EXTRA & SESSION_CERTS`.  I'll push a change that corrects `OPENSSL_ALL` to `OPENSSL_EXTRA` and adjusts the endif comment.